token/program-2022/src/extension/confidential_transfer/instruction.rs

```diff
@@ -228,16 +228,85 @@ pub enum ConfidentialTransferInstruction {
     ///   None
     ///
     DisableBalanceCredits,
+
+    /// Transfer all withheld confidential tokens in the mint to an account. Signed by the mint's
+    /// withdraw withheld tokens authority.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   0. `[writable]` The token mint. Must include the `TransferFeeConfig` extension.
+    ///   1. `[writable]` The fee receiver account. Must include the `TransferFeeAmount` and
+    ///      `ConfidentialTransferAccount` extensions.
+    ///   2. `[]` Instructions sysvar
+    ///   3. `[signer]` The mint's `withdraw_withheld_authority`.
+    /// or:
+    ///   3. `[]` The mint's `withdraw_withheld_authority`'s multisignature owner/delegate.
+    ///   4. ..3+M `[signer]` M signer accounts.
```

Comment:
Yes, I was following the existing syntax here and I don't have strong opinions on the "or" syntax or the "split" syntax. I am curious to hear what others think.

Reply:
@mvines Do you have any strong opinions on the style?

Reply:
I vote for consistency, so that sounds like the split style.  However let's do this in a follow-up PR. Imma waiting for this PR to land to unblock the test PR I'm working on.

Reply:
Sounds good. I'll convert it to a split style in a separate PR.